src/main/java/org/spongepowered/api/text/translation/Translation.java

```diff
@@ -24,19 +24,20 @@
  */
 package org.spongepowered.api.text.translation;
 
-import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.text.translation.locale.Locales;
+import java.util.Locale;
 
 /**
  * Represents an identifier for text that can be translated into multiple
- * languages. This is usually sent directly to the client and translated there,
- * not on the server.
+ * languages. Minecraft-included translations are generally translated clientside.
+ * Translations not included in Minecraft are generally expected to
+ * be translated server-side, for example using Gettext or a
+ * {@link java.util.ResourceBundle}
```

Comment:
mods with resource packs get translations from the GameRegistry. Those are clientside translations. Anything doing serverside translations uses a different Translation implementation appropriate to its translation store.